lumicks/pylake/force_calibration/power_spectrum.py

```diff
@@ -99,6 +100,105 @@ def _exclude_range(self, excluded_ranges) -> "PowerSpectrum":
         ps.power = ps.power[indices]
         return ps
 
+    def identify_peaks(
```

Comment:
Would also be nice to add a changelog entry for this one.

Reply:
Sure!